itou/templates/layout/breadcrumbs_from_dashboard.html

```diff
@@ -2,12 +2,7 @@
 <nav class="c-breadcrumb" aria-label="Fil d'Ariane">
     <ol class="breadcrumb">
         {# Common parts #}
-        <li class="breadcrumb-item">
-            <a href="{% url 'home:hp' %}">Accueil</a>
-        </li>
-        <li class="breadcrumb-item">
-            <a href="{% url 'dashboard:index' %}">Tableau de bord</a>
-        </li>
+        {% include "includes/homepage_breadcrumb.html" %}
```

Comment:
```suggestion
        {% include "includes/homepage_breadcrumb.html" with user=user only %}
```
Eventuellement (et pareil pour les autres includes de ce template) ?